pack/mts_encounter.json

```
@@ -717,6 +737,7 @@
 		"quantity": 1,
 		"set_code": "tower_defense",
 		"set_position": 15,
+		"text": "Attach to Proxima Midnght. \n [star]Proxima Midnight's attach gain overkill and piercing. \n <b> Hero Action: </b> Take 1 damage and spend [energy] [mental] resources → discard this card.",
```

The ⭐  text can be defined as:

```json
"attack": 1,
"attack_text": "Proxima Midnight's attacks gain overkill and piercing."
```

```suggestion
		"text": "Attach to Proxima Midnght.\n<b>Hero Action:</b> Take 1 damage and spend [energy] [mental] resources → discard this card.",
```